package.json

```diff
@@ -27,7 +27,7 @@
     "lint": "eslint . && jscs index.js bin/ lib/ test/",
     "prepublish": "marked-man --name gulp docs/CLI.md > gulp.1",
     "pretest": "npm run lint",
-    "test": "lab test/*.js -cv",
+    "test": "lab test/*.js -cv -I Reflect",
```

Comment:
yeah, this is annoying.  These tests will need to be switch to mocha like all the other repositories.  This is a fine fix for now.